.github/workflows/pylint.yaml

```diff
@@ -0,0 +1,41 @@
+name: pylint
+
+on:
+  pull_request:
+    branches: [ "2.0" ]
+
+jobs:
+  build:
+    name: pylint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install Pylint
+        run: pip install pylint
+
+      - name: Download previous run results
```

Comment:
Every run uploads the scan results that can be downloaded in the next run to be able to a comparison within runs.
Generated a message like this at the end of the scan:

```
------------------------------------------------------------------
Your code has been rated at 9.58/10 (previous run: 9.74/10, -0.16)
```

Reply:
is it going to comparer the results of the last run for the same branch/PR or for any of the PRs? (the latter would not make much sense right?)

Reply:
Yes we have to download .pylint.d of the latest commit on 2.0 branch

Reply:
yes thank you for noticing, I completely missed that. I will add modify it in the next commit